src/content/developers/docs/apis/json-rpc/index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: JSON RPC API
+description: A stateless, light-weight remote procedure call (RPC) protocol
+lang: en
+sidebar: true
+---
+
+The JSON-RPC specification allows for an application programming interface (API), similar to REST or GraphQL, with endpoints served from an Ethereum node. Several [JavaScript](/developers/docs/apis/javascript/#available-libraries) and [backend](/developers/docs/apis/backend/#available-libraries) libraries exist that conform to this specification, enabling developers to utilize up-to-date on-chain data while building [web3](/developers/docs/web2-vs-web3/) enabled [Dapps](/glossary/#dapp).
```

Comment:
This first sentence really confuses me.

the JSON RPC spec allows for an API with endpoints served from an ethereum node.

Does this mean it allows you to configure an API to call endpoints from an ethereum node?

I think the first sentence has too much going on to be clear. Once we've fixed this I think we can ship!